experimenter/experimenter/templates/common/experiment-details-sidebar.html

```diff
@@ -0,0 +1,146 @@
+<div x-data="{ redirectToReactRoute: function(slug, pageSlug) { window.location.href = `/nimbus/${slug}/edit/${pageSlug}`; } }"  class="flex  flex-col w-full h-full">
+  <div class="overflow-y-auto overflow-x-hidden flex-grow">
+      <div class="px-5">
+          <a class="flex justify-start space-x-2 items-center text-gray-500" href="{% url 'nimbus-detail' experiment.slug %}">
+            <i class="fa-solid fa-chevron-left h-3"></i>
+            <p class="text-md">Back to Summary</p>
+          </a>
+      </div>
+    <ul class="flex flex-col py-4 space-y-1">
+      <li class="px-5">
+        <div class="flex flex-row items-center h-8">
+          <div class="text-sm w-full  flex flex-row items-center justify-between font-light tracking-wide text-gray-500">
+              Edit
+              <hr class="border-t ml-2 w-full border-gray-200"/>
+          </div>
+        </div>
+      </li>
+      <li>
+        <button x-on:click="redirectToReactRoute('{{ experiment.slug }}', 'overview')"  
+        {% if experiment.can_edit %}
+          class="w-full flex flex-row items-center h-11 focus:outline-none hover:bg-gray-50 text-gray-700 hover:text-gray-800 border-l-4 border-transparent hover:border-[#b590d4] pr-6"
+        {% else %}
+          disabled
+          class="w-full flex flex-row items-center h-11  text-gray-400 grayscale cursor-not-allowed pr-6"
+        {% endif %}
+          >
+          <span class="inline-flex justify-center items-center ml-4">
+              <i class="fa-solid fa-gear"></i>
+          </span>
+          <span class="ml-2 text-sm tracking-wide truncate">Overview</span>
+        </button>
+      </li>
+      <li>
+        <button x-on:click="redirectToReactRoute('{{ experiment.slug }}', 'branches')" 
+        {% if experiment.can_edit %}
+        class="w-full flex flex-row items-center h-11 focus:outline-none hover:bg-gray-50 text-gray-700 hover:text-gray-800 border-l-4 border-transparent hover:border-[#b590d4] pr-6"
+      {% else %}
+        disabled
+        class="w-full flex flex-row items-center h-11  text-gray-400 grayscale cursor-not-allowed pr-6"
+      {% endif %}
+        >
+          <span class="inline-flex justify-center items-center ml-4">
+              <i class="fa-solid fa-layer-group"></i>
+          </span>
+          <span class="ml-2 text-sm tracking-wide truncate">Branches</span>
+        </button>
+      </li>
+      <li>
+        <button x-on:click="redirectToReactRoute('{{ experiment.slug }}', 'metrics')"  
+        {% if experiment.can_edit %}
+        class="w-full flex flex-row items-center h-11 focus:outline-none hover:bg-gray-50 text-gray-700 hover:text-gray-800 border-l-4 border-transparent hover:border-[#b590d4] pr-6"
+      {% else %}
+        disabled
+        class="w-full flex flex-row items-center h-11  text-gray-400 grayscale cursor-not-allowed pr-6"
+      {% endif %}
+        >
+          <span class="inline-flex justify-center items-center ml-4">
```

Comment:
is there any way we can make it reusable like components, I mean like overview, branch etc etc, define one component and reuse it here